### PR TITLE
Feataapulskis

### PR DESCRIPTION
**Pull request by ArvApu**

### CODE REVIEW

**Review by EdotJ — CHANGES_REQUESTED** [review]
just look at the last PR plz.

File: Web/Controllers/Investments/RealEstateController.cs [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            return PartialView("DeleteConfirmForm",
+                new DeletionViewModel {LtName = "nekilnojamas turtas", Id = id, Controller = "RealEstate", Method = "DeleteRealEstate"});
+        }
+        
+        [HttpDelete]
+        public IActionResult DeleteRealEstate(int id)
+        {
+            var realEstate = repository.RealEstate.FirstOrDefault(x => x.Id == id);
+        
+            repository.Remove(realEstate);
```
Comment (EdotJ) [inline Web/Controllers/Investments/RealEstateController.cs]: should be a separate method.

**Review by EdotJ — CHANGES_REQUESTED** [review]
change the method and you can merge.

File: Web/Controllers/Investments/RealEstateController.cs (hunk 1 above)
Comment (ArvApu) [reply Web/Controllers/Investments/RealEstateController.cs]: on it

Comment (ArvApu) [reply Web/Controllers/Investments/RealEstateController.cs]: perdairua i atskira metoda

**Review by EdotJ — APPROVED** [review]

**Pull request merged**